presto-orc/src/main/java/com/facebook/presto/orc/reader/NullWrappingColumnReader.java

```diff
@@ -66,7 +66,7 @@ protected void makeInnerQualifyingSet()
         int numActive = inputQualifyingSet.getPositionCount();
 
         innerQualifyingSet.reset(numActive);
-        int prevRow = posInRowGroup;
+        int prevRow = 0;
```

Comment:
I don't understand this change. posInRowGroup points to the first row in the row group that hasn't been read. If last scan exceeded the budget and now we need to re-read from the beginning, shouldn't posInRowGroup be reset to zero? I'm confused because here you replaced posInRowGroup with 0, but further down in this method the usage of posInRowGroup is unchanged. Would you clarify?